### ISSUE TITLE
Where is the global-project folder located in stack 2, and how to set global options?

### ISSUE THREAD

**Issue by OscarSouth** [commercialhaskell/stack#4995]
I used to previously pull in a ready made global-project with all my config and necessary self-made functionality (I'm using stack to configure my machines for live coding music performance, so it might not be great 'standard practice' for programming). When I tried to repeat this process with stack 2, the global project's presence seemed to have no effect. I also can't get the global ghc instance to build a specific version according to a stack.yaml file.

Basically none of the workflow described here is working for me with stack 2:
https://docs.haskellstack.org/en/stable/yaml_configuration/

Where can I find and modify the global-project using stack 2?

Cheers,
Oscar

**Comment by OscarSouth** [comment]
Follow up:
I've installed 1.9.3 and found the same behaviour (inside .stack/global-project, stack build builds correctly with ghc 8.2.2, outside it builds a fresh global versions with 8.6.5). This is behaviour that I've not seen before, though it is a new machine so there might be something I've not spotted. Any advice on this would be appreciated.
